tests/clients/test_kafka_python.py

```diff
@@ -0,0 +1,120 @@
+# (c) Copyright IBM Corp. 2025
+
+from typing import Generator
+
+import pytest
+from kafka import KafkaConsumer, KafkaProducer
+from kafka.admin import KafkaAdminClient, NewTopic
+from opentelemetry.trace import SpanKind
+
+from instana.singletons import agent, tracer
+from tests.helpers import testenv
+
+
+class TestKafkaPythonProducer:
+    @pytest.fixture(autouse=True)
+    def _setup(self) -> Generator[None, None, None]:
```

Comment:
```suggestion
    def _resource(self) -> Generator[None, None, None]:
```
We use _resource as the setup function name